src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -77,6 +88,18 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
   );
   const isSaboSubmitted = isReimbursementRequestSaboSubmitted(reimbursementRequest);
 
+  const {
```

Comment:
I would move all this form code and the modal into it's own file like we do for: `src\frontend\src\pages\FinancePage\FinanceComponents\ReportRefundModal.tsx`